src/parser/lexer.js

```diff
@@ -0,0 +1,21 @@
+const Token = require('./token');
+class Lexer {
+  static tokenize(input) {
+    const stringTokens = input
+      .split(/( )|(:)|(")/g)
+      .filter(text => text);
+    return stringTokens.map(s => {
+      if (s === `:`)
+        return new Token(`colon`, s);
+      if (s === `"`)
+        return new Token(`quote`, s);
+      if (s === ` `)
+        return new Token(`space`, s);
+      if (s === `or`)
```

Comment:
It seems this will have trouble if you encounter a string with "` or `" in it, like `Fact or Fiction`? It's probably best to use a different symbol.